nmdc_server/api.py

```diff
@@ -746,6 +751,33 @@ async def update_submission(
     crud.update_submission_lock(db, submission.id)
     return submission
 
+def create_github_issue(submission,user):
+
+    url = "https://api.github.com/repos/JamesTessmer/Issue-bot-testing/issues?state=all"
```

Comment:
That's a good idea. I was trying to figure out the best way to set both of those because they definitely shouldn't be hard coded in. I'll implement this and the check.

Reply:
Added this change today.